cmd/kpromo/cmd/pr/pr.go

```diff
@@ -316,33 +355,35 @@ func runPromote(opts *promoteOptions) error {
 	}
 
 	// Push to fork
-	if mustRun(opts, fmt.Sprintf("Push changes to user's fork at %s/%s?", userForkOrg, userForkRepo)) {
-		logrus.Infof("Pushing manifest changes to %s/%s", userForkOrg, userForkRepo)
+	if mustRun(opts, fmt.Sprintf("Push changes to user's fork at %s/%s?", opts.userFork, k8sioRepo)) {
+		logrus.Infof("Pushing manifest changes to %s/%s", opts.userFork, k8sioRepo)
 		if err := repo.PushToRemote(github.UserForkName, branchname); err != nil {
-			return fmt.Errorf("pushing %s to %s/%s: %w", github.UserForkName, userForkOrg, userForkRepo, err)
+			return fmt.Errorf("pushing %s to %s/%s: %w", github.UserForkName, opts.userFork, k8sioRepo, err)
 		}
 	} else {
 		// Exit if no push was made
-
-		logrus.Infof("Exiting without creating a PR since changes were not pushed to %s/%s", userForkOrg, userForkRepo)
+		logrus.Infof("Exiting without creating a PR since changes were not pushed to %s/%s", opts.userFork, k8sioRepo)
 		return nil
 	}
 
-	// Create the Pull Request
-	if mustRun(opts, "Create pull request?") {
-		pr, err := gh.CreatePullRequest(
-			git.DefaultGithubOrg, k8sioRepo, k8sioDefaultBranch,
-			fmt.Sprintf("%s:%s", userForkOrg, branchname),
-			commitMessage, generatePRBody(opts),
-		)
-		if err != nil {
-			return fmt.Errorf("creating the pull request in k/k8s.io: %w", err)
-		}
-		logrus.Infof(
-			"Successfully created PR: %s%s/%s/pull/%d",
-			github.GitHubURL, git.DefaultGithubOrg, k8sioRepo, pr.GetNumber(),
-		)
+	return nil
+}
+
+func createPullRequest(opts *promoteOptions, branchname string) error {
+	gh := github.New()
+	pr, err := gh.CreatePullRequest(
+		git.DefaultGithubOrg, k8sioRepo, k8sioDefaultBranch,
+		fmt.Sprintf("%s:%s", opts.userFork, branchname),
+		"Image promotion for "+opts.project+" "+strings.Join(opts.tags, " / "),
```

Comment:
This doesn't match the original behavior, the commit message can be extended in some case, e.g. with the `releng:` prefix.